### PR TITLE
RHCLOUD-33713 adds grafana dashboards

### PR DESCRIPTION
**Pull request by tonytheleg**
## Describe your changes

- adds dashboards for relations-api and spicedb

## Ticket reference (if applicable)
For RHCLOUD-33713 and RHCLOUD-33714

## Checklist

* [ ] Are the agreed upon acceptance criteria fulfilled?

* [ ] Was the 4-eye-principle applied? (async PR review, pairing, ensembling)

* [ ] Do your changes have passing automated tests and sufficient observability?

* [ ] Are the work steps you introduced repeatable by others, either through automation or documentation?
  * [ ] If automation is possible but not done due to other constraints, a ticket to the tech debt sprint is added
  * [ ] An SOP (Standard Operating Procedure) was created

* [ ] The Changes were automatically built, tested, and  - if needed, behind a feature flag - deployed to our production environment. (**Please check this when the new deployment is done and you could verify it.**)

* [ ] Are the agreed upon coding/architectural practices applied?

* [ ] Are security needs fullfilled? (e.g. no internal URL)

* [ ] Is the corresponding Ticket in the right state? (should be on "review" now, put to done when this change made it to production)

* [ ] For changes to the public API / code dependencies: Was the whole team (or a sufficient amount of ppl) able to review?

### CODE REVIEW

**Comment by gwenneg** [comment]
Sorry, I'm using your PR to test something in Konflux :sweat_smile: I doesn't actually need to be retested.

**Comment by gwenneg** [comment]
`/retest` could not re-trigger the pipeline for some reason... Nothing actually failed. The pipeline just did not start. Please ignore everything I did in this PR!

File: dashboards/grafana-dashboard-kessel-relations-api.configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,737 @@
...
+      },
+      "time": {
+        "from": "now-5m",
+        "to": "now"
+      },
+      "timepicker": {},
+      "timezone": "browser",
+      "title": "Kessel - Relations API",
+      "uid": "ddxs3i6o0xpmoe",
+      "version": 6,
```
Comment (Rajagopalan-Ranganathan) [inline dashboards/grafana-dashboard-kessel-relations-api.configmap.yaml:728]: better to start with version 1? WDYT?

File: dashboards/grafana-dashboard-kessel-spicedb.configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,4702 @@
...
+          "24h",
+          "2d",
+          "7d",
+          "30d"
+        ]
+      },
+      "timezone": "",
+      "title": "Kessel - SpiceDB ",
+      "uid": "gn38yKZnk-import",
+      "version": 19,
```
Comment (Rajagopalan-Ranganathan) [inline dashboards/grafana-dashboard-kessel-spicedb.configmap.yaml:4692]: version 1 may be?

Comment (tonytheleg) [reply dashboards/grafana-dashboard-kessel-spicedb.configmap.yaml:4692]: Ahh good call, thats what i get for copy/paste!

**Pull request merged**